pkg/virt-controller/services/template.go

```diff
@@ -907,6 +907,7 @@ func (t *templateService) RenderHotplugAttachmentPodTemplate(volumes []*v1.Volum
 					},
 				},
 			},
+			Tolerations: ownerPod.Spec.Tolerations,
```

Comment:
Do we want to do a DeepCopy here, I have had some instances where not doing the deep copy would produce unexpected results if the struct was later modified somewhere else.

Reply:
Get，done.